plugins/plugin-svn/che-plugin-svn-ext-ide/src/main/java/org/eclipse/che/plugin/svn/ide/commit/CommitPresenter.java

```diff
@@ -188,18 +195,44 @@ public void showDiff(String path) {
 
         checkState(project != null);
 
-        service.showDiff(project.getLocation(), new Path[]{valueOf(path)}, "HEAD").then(new Operation<CLIOutputResponse>() {
-            @Override
-            public void apply(CLIOutputResponse response) throws OperationException {
-                String content = Joiner.on('\n').join(response.getOutput());
-                diffViewerPresenter.showDiff(content);
-            }
-        }).catchError(new Operation<PromiseError>() {
-            @Override
-            public void apply(PromiseError error) throws OperationException {
-                notificationManager.notify(error.getMessage(), FAIL, FLOAT_MODE);
-            }
-        });
+        showDiff(path, project, null, null);
+    }
+
+    private void showDiff(final String path, final Project project, final String userName, final String password) {
+        service.showDiff(project.getLocation(),
+                         new Path[]{valueOf(path)},
+                         "HEAD",
+                         userName,
+                         password)
+               .then(new Operation<CLIOutputResponse>() {
+                   @Override
+                   public void apply(CLIOutputResponse response) throws OperationException {
+                       String content = Joiner.on('\n').join(response.getOutput());
+                       diffViewerPresenter.showDiff(content);
+                   }
+               })
+               .catchError(new Operation<PromiseError>() {
+                   @Override
+                   public void apply(PromiseError error) throws OperationException {
+                       if (getErrorCode(error.getCause()) == UNAUTHORIZED_SVN_OPERATION) {
```

Comment:
In which cases here will be UNAUTHORIZED_SVN_OPERATION? Only when user tries to perform operation without credentials?